libtock/ambient_light.c

```diff
@@ -38,20 +38,10 @@ int ambient_light_read_intensity_sync(int* lux_value) {
 }
 
 int ambient_light_subscribe(subscribe_cb callback, void* userdata) {
-  subscribe_return_t ret = subscribe2(DRIVER_NUM_AMBIENT_LIGHT, 0, callback, userdata);
-  if (ret.success) {
-    return TOCK_SUCCESS;
-  } else {
-    return tock_error_to_rcode(ret.error);
-  }
+  return subscribe(DRIVER_NUM_AMBIENT_LIGHT, 0, callback, userdata);
 }
 
 int ambient_light_start_intensity_reading(void) {
-  syscall_return_t ret = command2(DRIVER_NUM_AMBIENT_LIGHT, 1, 0, 0);
-  if (ret.type == TOCK_SYSCALL_SUCCESS) {
-    return TOCK_SUCCESS;
-  } else {
-    return tock_error_to_rcode(ret.data[0]);
-  }
+  return command(DRIVER_NUM_AMBIENT_LIGHT, 1, 0, 0);
```

Comment:
This seems to revert the ambient light --> 2.0 changes, and should probably be removed?

Reply:
Hm, yeah. I suspect this is an uncrustify thing.